Bearded.Utilities.Tests/Geometry/CircularArc2Tests.cs

```diff
@@ -10,354 +10,332 @@ namespace Bearded.Utilities.Tests.Geometry
 {
     public sealed class CircularArc2Tests
     {
-        public sealed class ShortArcBetweenDirections
+        private const float epsilon = 0.001f;
+
+        [Property(Arbitrary = new[] { typeof(DirectionGenerators.All) })]
```

Comment:
May be more convenient to register the direction generator that's used everywhere just once in the constructor? (see https://github.com/beardgame/td/blob/develop/src/Bearded.TD.Tests/Game/Generation/AreaTagsTests.cs )

Reply:
Didn't know about that trick, thanks!